lib/puppet/provider/network_vlan/ios.rb

```diff
@@ -76,8 +76,17 @@ def create(context, name, should)
       update(context, name, should)
     end
 
+    def self.check_vlan_can_be_deleted(name)
+      if ['1', '1002', '1003', '1004', '1005'].include?(name)
+        warn "Vlan #{name} is a Cisco default vlan and may not be deleted."
```

Comment:
Total nit - but the message should be VLAN - right now it's mixed case of Vlan and vlan.

Reply:
Good call, worth getting case right, I've changed these to VLAN